assets/content/static/navbar.html

```diff
@@ -79,7 +79,7 @@ <h6 class="text-primary mb-md-1">AcadeMix</h6>
                 </li>
 
                 <li class="nav-item dropdown">
-                    <a href="https://blog.sefglobal.org" class="nav-link" role="button">
+                    <a href="https://https://sefglobal.medium.com/" class="nav-link" role="button">
```

Comment:
You have mistakenly added another `https://` to the link

Reply:
You can fix this issue in another commit in the same branch and push. So the PR will be updated automatically. No need to close the PR and send another one.